src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {
+  memo, useEffect, useMemo, useState,
+} from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../components/Loader';
+import { PersonTable } from '../components/PersonTable/PersonTable';
+import { Person } from '../types';
+
+export const PeoplePage: React.FC = memo(() => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const { slug } = useParams();
+  const [errorText, setErrorText] = useState('');
+
+  const errorMessage = (error: string) => {
+    setErrorText(error);
+  };
+
+  const peopleFromServer = useMemo(() => async () => {
```

Comment:
It's better to use `useCallback
```suggestion
  const getPeopleFromServer = useCallback(async () => {
```